package.json

```diff
@@ -85,26 +85,22 @@
     "@babel/plugin-proposal-object-rest-spread": "^7.3.2",
     "@babel/plugin-transform-template-literals": "^7.4.4",
     "@babel/preset-env": "^7.4.5",
-    "babel-loader": "^8.0.5",
-    "bundlesize": "^0.17.2",
+    "bundlesize": "^0.18.0",
     "dotenv": "^8.1.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-config-prettier": "^4.0.0",
+    "eslint": "^6.5.1",
+    "eslint-config-airbnb-base": "^14.0.0",
```

Comment:
You'll see here we were using `eslint-config-airbnb` which includes a bunch of React stuff and requires us to also install `eslint-plugin-react`, `eslint-plugin-jsx-a11y`, and, most recently, `eslint-plugin-react-hooks`. We're not actually linting react files (even the ones in sandbox), so we can use `eslint-config-airbnb-base` which doesn't require any of the react linting stuff.